src/test/java/dtu/scheduler/LoginLogoutSteps.java

```diff
@@ -8,15 +8,22 @@
 //Philip Hviid
 public class LoginLogoutSteps {
 	private String workerId;
-	private SchedulingApp app;
+	private SchedulingApp schedulingApp;
 	private ErrorMessageHolder msg;
-	private WorkerDAO workerDAO;
+
 	
-	public LoginLogoutSteps(SchedulingApp app, ErrorMessageHolder msg, WorkerDAO  workerDAO) {
-		this.app = app;
+	public LoginLogoutSteps(SchedulingApp app, ErrorMessageHolder msg) {
+		this.schedulingApp = app;
 		this.msg = msg;
-		this.workerDAO = workerDAO;
 	}
+	
+	@Given("that a worker is logged in")
+	public void that_a_worker_is_logged_in() throws WorkerDoesNotExistException {
+	    schedulingApp.logIn("ASDF");
+	    schedulingApp.getCurrentUser();
```

Comment:
I'm guessing .getCurrentUser() throws an exception, so you run it to verify. I think it would be more "clean" code to assert that the user you get back isn't null, rather than throw an exception in the first place, it feels bad to get a value and never use it

Reply:
The exception throw is to account for logIn(), in case you try to log in with a signature that does not exist in the Database, and not related to getCurrentUser().